pkg/liqo-controller-manager/external-network/client-operator/client_controller.go

```diff
@@ -76,8 +76,8 @@ func NewClientReconciler(cl client.Client, dynClient dynamic.Interface,
 
 // Reconcile manage GatewayClient lifecycle.
 func (r *ClientReconciler) Reconcile(ctx context.Context, req ctrl.Request) (res ctrl.Result, err error) {
-	gwClient := &networkingv1alpha1.GatewayClient{}
-	if err = r.Get(ctx, req.NamespacedName, gwClient); err != nil {
+	client := &networkingv1alpha1.GatewayClient{}
```

Comment:
Can we keep the gwClient name? Client can be misleading

Reply:
If we refactor to `gwClient`, I would also refactor the server_controller.go file using `gwServer` to be coherent and to ease spotting differences between the two files.